cv_origami/exceptions.py

```diff
@@ -0,0 +1,23 @@
+class OrigamiException(Exception):
+    """
+    Base class for all exceptions thrown by origami.
+    """
+    STATUS_CODE = 1
+
+    def __init__(self, message=''):
+        super().__init__("OrigamiException[{0}] => {1}".format(
+            self.STATUS_CODE, message))
+
+
+class OrigamiConfigException(OrigamiException):
+    """
+    Bundled zip provided by user or by server is not valid.
+    """
+    STATUS_CODE = 100
+
+
+class InvalidDemoBundleException(OrigamiException):
+    """
+    Bundled zip provided by user or by server is not valid.
+    """
+    STATUS_CODE = 200
```

Comment:
Why 200 on Invalid zip?

Reply:
These exceptions are internal to the package. They doesn't signify anything, I only tried to club similar kind of exception under similar status code as 2xx for DemoBundle related exception like this one and `DemoDeployFailureException`

Reply:
How are you planning to inform the user that his/her bundle was invalid?